deploy

```diff
@@ -27,8 +27,9 @@ elif [[ "${TRAVIS_BRANCH}" =~ $ITB_BRANCH_REGEXP ]]; then
     mkdir -p $CSS_DIR
     # If we set the colors in extra.css, replace them with the ITB colors.
     # Otherwise, copy over the ITB color CSS.
-    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css || cp ci/itb.css $CSS_DIR
+    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css ||:
```

Comment:
This is better, but to me the `||:` things still seem a bit odd.  Their effect is, "`set -e` is in effect, but nevertheless don't exit the script early if these `sed` commands error out when, eg, `extra.css` does not exist."

Is that what you want?  That is, `extra.css` being missing is OK?

Reply:
Yeah, it's fine if it doesn't exist, copying in `itb.css` ensures that we have the right config in place.

That being said, maybe I should just wrap the seds in `set +e...set -e`

Reply:
Nah it's fine to use `|| :` as long as that's the semantics you want.